software/teachee-desktop/src/app.rs

```diff
@@ -10,13 +10,43 @@ enum Channel {
     Cos,
 }
 
+#[derive(Debug, Default)]
+enum TriggerControl {
+    #[default]
+    Start,
+    Stop,
+}
+
+// Allows enum to be formatted to string
+impl fmt::Display for TriggerControl {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        fmt::Debug::fmt(self, f)
+    }
+}
+
+const GROUP_SPACING: f32 = 3.0;
+const BUTTON_HEIGHT: f32 = 25.0;
+
 #[derive(Debug, Default)]
 pub struct App {
     flag: bool,
     channel1: Channel,
     channel1_offset: f64,
     channel2: Channel,
     channel2_offset: f64,
```

Comment:
While you are working with this code could you also change these fields to be an array?

Reply:
I think it's best we do this in a new PR since this one has enough changes already. I will make a new PR after this is merged to clean up the old plot code